From: Iain Lane <laney@debian.org>
Date: Tue, 3 Sep 2019 18:16:10 +0200
Subject: Revert "build: Include libdir in rpath"

This reverts commit 6a9199a86a7047aa3d5db8d0faba8a5ae1e10bf4.

We don't want these in rpath on Debian; the standard libdir is in the
search path.

Forwarded: no, Debian-specific
---
 meson.build                       | 1 -
 src/miners/fs/meson.build         | 2 +-
 src/miners/rss/meson.build        | 2 +-
 src/tracker-control/meson.build   | 2 +-
 src/tracker-extract/meson.build   | 2 +-
 src/tracker-writeback/meson.build | 2 +-
 src/tracker/meson.build           | 2 +-
 7 files changed, 6 insertions(+), 7 deletions(-)

diff --git a/meson.build b/meson.build
index 8874973..08b0c78 100644
--- a/meson.build
+++ b/meson.build
@@ -420,7 +420,6 @@ build_root = meson.current_build_dir()
 # directory, following the existing convention in that directory.
 tracker_api_versioned_name = 'tracker-miners-@0@'.format(tracker_api_version)
 tracker_internal_libs_dir = join_paths(get_option('prefix'), get_option('libdir'), tracker_api_versioned_name)
-tracker_install_rpath = ':'.join([tracker_internal_libs_dir, libdir])
 
 tracker_extract_modules_dir = join_paths(get_option('prefix'), get_option('libdir'), tracker_api_versioned_name, 'extract-modules')
 tracker_writeback_modules_dir = join_paths(get_option('prefix'), get_option('libdir'), tracker_api_versioned_name, 'writeback-modules')
diff --git a/src/miners/fs/meson.build b/src/miners/fs/meson.build
index e700dee..84326be 100644
--- a/src/miners/fs/meson.build
+++ b/src/miners/fs/meson.build
@@ -26,7 +26,7 @@ executable('tracker-miner-fs-@0@'.format(tracker_api_major),
     c_args: tracker_c_args,
     install: true,
     install_dir: get_option('libexecdir'),
-    install_rpath: tracker_install_rpath,
+    install_rpath: tracker_internal_libs_dir,
 )
 
 # This file allows starting the service with D-Bus autoactivation.
diff --git a/src/miners/rss/meson.build b/src/miners/rss/meson.build
index 84eaa6b..51af115 100644
--- a/src/miners/rss/meson.build
+++ b/src/miners/rss/meson.build
@@ -9,7 +9,7 @@ executable('tracker-miner-rss-@0@'.format(tracker_api_major),
     c_args: tracker_c_args,
     install: true,
     install_dir: get_option('libexecdir'),
-    install_rpath: tracker_install_rpath,
+    install_rpath: tracker_internal_libs_dir,
 )
 
 # This file allows starting the service with D-Bus autoactivation.
diff --git a/src/tracker-control/meson.build b/src/tracker-control/meson.build
index 9270350..1a2a352 100644
--- a/src/tracker-control/meson.build
+++ b/src/tracker-control/meson.build
@@ -36,7 +36,7 @@ executable('tracker-miner-fs-control-@0@'.format(tracker_api_major),
     c_args: tracker_c_args,
     install: true,
     install_dir: get_option('libexecdir'),
-    install_rpath: tracker_install_rpath,
+    install_rpath: tracker_internal_libs_dir,
 )
 
 # This file allows starting the service with D-Bus autoactivation.
diff --git a/src/tracker-extract/meson.build b/src/tracker-extract/meson.build
index ebb6b16..88ce49f 100644
--- a/src/tracker-extract/meson.build
+++ b/src/tracker-extract/meson.build
@@ -162,7 +162,7 @@ executable('tracker-extract-@0@'.format(tracker_api_major),
   dependencies: tracker_extract_dependencies,
   install: true,
   install_dir: join_paths(get_option('prefix'), get_option('libexecdir')),
-  install_rpath: tracker_install_rpath)
+  install_rpath: tracker_internal_libs_dir)
 
 # This file allows starting the service with D-Bus autoactivation.
 # It's also used by tracker-control to identify the miner.
diff --git a/src/tracker-writeback/meson.build b/src/tracker-writeback/meson.build
index 321e149..04ac38c 100644
--- a/src/tracker-writeback/meson.build
+++ b/src/tracker-writeback/meson.build
@@ -45,7 +45,7 @@ executable('tracker-writeback-@0@'.format(tracker_api_major),
   dependencies: [gmodule, tracker_miner, tracker_sparql, tracker_miners_common_dep],
   install: true,
   install_dir: join_paths(get_option('prefix'), get_option('libexecdir')),
-  install_rpath: tracker_install_rpath)
+  install_rpath: tracker_internal_libs_dir)
 
 dbus_service_file = configure_file(
   input: 'org.freedesktop.Tracker3.Writeback.service.in',
diff --git a/src/tracker/meson.build b/src/tracker/meson.build
index 1433f83..368f34d 100644
--- a/src/tracker/meson.build
+++ b/src/tracker/meson.build
@@ -24,7 +24,7 @@ foreach m: modules
             '-DMANDIR="@0@"'.format(join_paths(get_option('prefix'), get_option('datadir'), 'man')),
         ],
         install: true,
-        install_rpath: tracker_install_rpath,
+        install_rpath: tracker_internal_libs_dir,
         install_dir: join_paths(get_option('libexecdir'), 'tracker@0@'.format(tracker_api_major)),
         dependencies: [tracker_sparql, tracker_miners_common_dep],
         include_directories: [commoninc, configinc, srcinc],
